src/client/components/Calandar/ControlSlot.js

```diff
@@ -0,0 +1,3 @@
+import createSlot from 'react-tackle-box/Slot'
```

Comment:
What does this file do? I don't think it is used anywhere

Reply:
Ping

Reply:
Also, this is a js file for some reason?? But pretty sure it should just be deleted